dev/com.ibm.ws.concurrent.mp_fat_jakarta/fat/src/com/ibm/ws/concurrent/mp/fat/jakarta/MPContextProp1_3_EE10_Test.java

```diff
@@ -16,28 +16,24 @@
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import com.ibm.websphere.simplicity.ShrinkHelper;
 
 import componenttest.annotation.MinimumJavaLevel;
 import componenttest.annotation.Server;
-import componenttest.annotation.TestServlet;
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
-import concurrent.mp.fat.v13.ee10.web.MPContextProp1_3_EE10_TestServlet;
 
-// TODO This temporarily runs with MP Context Propagation 1.3 because 2.0 isn't available yet.
-//      See the comments in FATSuite.java
 @RunWith(FATRunner.class)
 @MinimumJavaLevel(javaLevel = 11)
 public class MPContextProp1_3_EE10_Test extends FATServletClient {
```

Comment:
I think the issue here is that even though the test class has the `@MinimumJavaLevel` annotation junit processes tests by creating a list of test methods, and then filtering them out. Our custom FATRunner uses the `@TestServlet` annotation to grab the servlet class and find additional test methods. 

I did some digging and I think the solution would be to add the `@MinimumJavaLevel(javaLevel = 11)` to the servlet class which also contains `@Test` annotations. 

Your solution also works, but I think the additional annotation change might be the actual solution here.  This obviously doesn't work if two test classes share the same servlet, and that might be a situation for the build team to look at.